src/sections/Testimonies.tsx

```diff
@@ -116,16 +151,17 @@ export const Testimonies = () => (
       <Br />
       Découvrez les applications stars de La Suite :
     </p>
-    <div className="hidden md:flex flex-row gap-7">
-      {data.map((testimony) => (
-        <Card {...testimony} key={testimony.key} />
-      ))}
-    </div>
-    <div className="w-full md:hidden">
+    <div className="w-full">
       <SwiperWrapper
         slides={data.map((testimony) => (
           <Card {...testimony} key={testimony.key} />
         ))}
+        breakpoints={{
+          1024: {
+            slidesPerView: 3,
+            spaceBetween: 28,
+          },
```

Comment:
Sadly we can't rely on our tailwind tokens with swiper here, must use pixel values.